### PR TITLE
Examples: Make Mirror examples resizable.

### PR DESCRIPTION
**Pull request by gonnavis**
Make `webgl_mirror.html` and `webgl_mirror_nodes.html` resizable and add stats to them.

Make `webgl_mirror_nodes.html` consider `devicePixelRatio` to solve the problem of aliasing on high-pixelRaito devices.

https://raw.githack.com/gonnavis/three.js/ExamplesMirrorResizable/examples/webgl_mirror.html

https://raw.githack.com/gonnavis/three.js/ExamplesMirrorResizable/examples/webgl_mirror_nodes.html

### CODE REVIEW

File: examples/webgl_mirror.html [hunk 1]
```diff
@@ -26,14 +26,11 @@
 
 			import { OrbitControls } from './jsm/controls/OrbitControls.js';
 			import { Reflector } from './jsm/objects/Reflector.js';
-
-			// scene size
-			const WIDTH = window.innerWidth;
-			const HEIGHT = window.innerHeight;
+			import Stats from "./jsm/libs/stats.module.js";
```
Comment (Mugen87) [inline examples/webgl_mirror.html:29]: I don't think it's necessary to add `stats.js` to both examples.

File: examples/webgl_mirror.html [hunk 2]
```diff
@@ -43,6 +40,9 @@
 
 			let sphereGroup, smallSphere;
 
+			let groundMirrorRenderTarget, verticalMirrorRenderTarget;
```
Comment (Mugen87) [inline examples/webgl_mirror.html:43]: There is no need to cache the render targets in these variables. Just request them with `getRenderTarget()` in `onWindowResize()`.

**Comment by gonnavis** [comment]
@Mugen87 Updated.

**Comment by Mugen87** [comment]
> Make webgl_mirror_nodes.html consider devicePixelRatio to solve the problem of aliasing on high-pixelRaito devices.

In this case, `BlurNode` also has to honor the `devicePixelRatio`.

**Comment by gonnavis** [comment]
OK, made `devicePixelRatio` consider `BlurNode`.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**